### ISSUE TITLE
Can't remove chart in case of 'no data' 

### ISSUE THREAD

**Issue by preimann01** [codership/galera-manager-support#20]
Hi,
accidentally I've created a chart that does not have any data:

<img width="475" alt="Bildschirmfoto" src="https://user-images.githubusercontent.com/83397438/148178599-828aee3b-03ad-40bc-9650-d92737e2788b.png">

Now I can't remove it. All other charts which have data are removable. 
How can I remove the defective chart?

**Comment by mihaellenic** [comment]
Could you download the dashboard configuration and post it here? (Click on three dots in the top right corner of the dashboard, and you should see `Download dashboard configuration` action there)

We did some changes to the dashboard that will be available in the next release, and this issue should be fixed, but it would help us know for sure if we can test with your dashboard config.

**Comment by preimann01** [comment]
```JSON
{"name":xxx","config":{"charts":[{"id":"xxx1","title":"wsrep_received","metric":{"name":"wsrep_received","table":"mysql"},"position":1,"resolution":"5s","aggregation":"differential","unit":"tx","dataScaling":1},{"id":"xxx2","title":"wsrep_replicated","metric":{"name":"wsrep_replicated","table":"mysql"},"position":2,"resolution":"5s","aggregation":"differential","unit":"tx","dataScaling":1},{"id":"xxx3","title":"cpu","metric":{"name":"cpu_usage","table":"procstat"},"position":0,"resolution":"5s","aggregation":"raw","unit":"%","dataScaling":1,"yAxisMax":100},{"id":"xxx4","title":"wsrep_flow_control_sent","metric":{"name":"wsrep_flow_control_sent","table":"mysql"},"position":4,"resolution":"5s","aggregation":"differential","unit":"","dataScaling":1},{"id":"xxx5","title":"wsrep_flow_control_paused","metric":{"name":"wsrep_flow_control_paused","table":"mysql"},"position":3,"resolution":"5s","aggregation":"differential","unit":"","dataScaling":1},{"id":"xxx6","position":5,"resolution":"5s","title":"wsrep_replicated_bytes","metric":{"name":"wsrep_replicated_bytes","table":"mysql"},"aggregation":"raw","dataScaling":1,"unit":"Unit","yAxisMax":4},{"id":"xxx7","position":6,"resolution":"5s","title":"access denied","metric":{"name":"access_denied_errors","table":"mysql"},"aggregation":"raw","dataScaling":1},{"id":"xxx8","position":7,"resolution":"5s","title":"innodb_buffer_pool_bytes_data","metric":{"name":"innodb_buffer_pool_bytes_data","table":"mysql"},"aggregation":"raw","dataScaling":1}],"tileSize":"sm","refreshInterval":5000,"period":"15m"}}
```
The id xxx7 is the one which I can't remove.

**Comment by preimann01** [comment]
As far as I can see in the HTML source code the div which can't removed has an additional div with the "no data" content in it. May be this div prevents from reaching the unerlying div with the plotly svg?
As soon as I remove this "no data" div (by browser tools) it is possible to remove the chart . Unfortunately it leaves a space where the chart was seen before. Inside the dashboard config position6 isn't available anymore, but position7 remains and is not the new position6 as I would expect.

**Comment by mihaellenic** [comment]
Thank you for the dashboard config and the additional info. There seems to be nothing wrong with the configuration itself.

> As far as I can see in the HTML source code the div which can't removed has an additional div with the "no data" content in it. May be this div prevents from reaching the unerlying div with the plotly svg?

You are correct, that's the issue. And it's already fixed and will be available in the next release which is coming soon.

**Comment by mihaellenic** [comment]
> As soon as I remove this "no data" div (by browser tools) it is possible to remove the chart . Unfortunately it leaves a space where the chart was seen before. Inside the dashboard config position6 isn't available anymore, but position7 remains and is not the new position6 as I would expect.

Currently, the charts will only fill vertical empty space, meaning that the chart will move up if you remove the one above. But we will consider your feedback, it does sound reasonable.

**Comment by denisgcm** [comment]
The problem addressed in the issue should be now solved in the Galera Manager version 1.5.0.
